catalog/app/containers/Bucket/PackageDialog/PackageCreationForm.tsx

```diff
@@ -74,6 +74,8 @@ function filesStateToEntries(files: FI.FilesState): PD.ValidationEntry[] {
     R.mergeLeft(files.added, files.existing),
     R.omit(Object.keys(files.deleted)),
     Object.entries,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    R.filter(([path, file]) => !path.endsWith(FI.EMPTY_DIR_MARKER)),
```

Comment:
you can make it `[path]` or `[path,]` to avoid the lint error

Reply:
In both cases, typescript fails to infer type